requirements.txt

```diff
@@ -1,8 +1,6 @@
-mkdocs-material
-mkdocs-bootstrap4
-mkdocs-awesome-pages-plugin
```

Comment:
README for `mkdocs-pagenav-generator` says `mkdocs-awesome-pages-plugin` is required. Since the former isn't actively maintained, it would be safer to freeze it to avoid breakage in the future.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/36/commits/cebc4d98b191cfb43da8526f61d8da465cf711c9
Also added `awesome-pages` back to plugins as per `mkdocs-pagenav-generator` README it is required.